tests/test_openapi.py

```diff
@@ -93,93 +95,206 @@ def get_oas_30_(config):
     return get_oas_30(config)
 
 
-def test_cql_filters(get_oas_30_, config):
+@pytest.fixture()
+def is_cql(config):
     """
-    Added assertions CQL extension in OpenAPI 3.0 Document
+    Checks whether any resource supports feature filter
 
     :param config: configuration object
-    :param config: OpenAPI 3.0 Document object
 
+    :returns: Boolean value
     """
-    assert isinstance(config, dict)
-
-    cql_filter_exists = False
-
     collections = filter_dict_by_key_value(config['resources'],
                                            'type', 'collection')
+    for k, _ in collections.items():
+        if 'filters' in collections[k]:
+            return True
+    return False
 
-    assert isinstance(get_oas_30_, dict)
 
-    # assertion for get paths
+@pytest.fixture()
+def cql_components(get_oas_30_):
```

Comment:
I don’t think it makes sense to split the configuration into different fixtures. We agreed to split just the assertions into several functions for unit tests

Reply:
Depending on the structure of unit test functions I tried to combine the configurations as much as I can. https://github.com/geopython/pygeoapi/pull/478/commits/27c7c7794117e558fe5f01a99e23ec9d3f35e628 for your reference.